install.sh

```diff
@@ -1,10 +1,12 @@
 sudo apt update
-sudo apt install -y python3-pip git python3-dev python3-libgpiod ffmpeg
+sudo apt install -y python3-pip git python3-dev python3-libgpiod ffmpeg maven
 git clone https://github.com/CornellCollegeComputingClub/c4-sign
 cd c4-sign
 git config pull.rebase false
 sudo -H python3 -m venv ./venv
 sudo -H /home/c4/c4-sign/venv/bin/python3 -m pip install -e '.[physical]'
 sudo cp -v ./bin/c4-sign.service /lib/systemd/system/
+./bin/setup_java_project.sh
+./bin/compile_java_project.sh
```

Comment:
so we run compile java project here, which is fine, but what about on sign startup (like if people update their code or add new code)?

Reply:
Do you mean in `bin/start_sign.sh`? I should have done that.

I think that when the project is in development on the developer's machine, I would like to keep compiling as a manual step, maybe.

Reply:
I've added a compile step to both the `install.sh` script and to the `start_sign.sh` script! I think...